src/main/java/uk/gov/mca/beacons/service/configuration/SecurityConfiguration.java

```diff
@@ -18,9 +18,7 @@ protected void configure(HttpSecurity http) throws Exception {
   }
 
   @Override
-  public void configure(WebSecurity web) throws Exception {
-    web
-      .ignoring()
-      .antMatchers("/registrations/**", "/actuator/health", "/actuator/info");
```

Comment:
Didn't know that there was a "/actuator/info" endpoint, but was it removed from here for a reason?

Reply:
Great catch!  Just put `/actuator/info` back in 👍🏻